h2o-py/tests/testdir_misc/pyunit_showtime.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import print_function
```

Comment:
This is not needed. 
Instead, go to your `h2o-3/h2o-py` folder and execute `pip install -e .`